qiita_pet/static/js/sharing.js

```diff
@@ -1,4 +1,4 @@
-var current_study = null;
+var current_id = null;
```

Comment:
This was discussed with @ElDeveloper in the last PR and it is not feasible to do at this time.